clients/ops/admin-ui/src/features/datastore-connections/add-connection/ConfigureConnector.tsx

```diff
@@ -1,52 +1,73 @@
 import { Flex } from "@fidesui/react";
-import { ConnectionOption } from "connection-type/types";
-import { SystemType } from "datastore-connections/constants";
-import React, { useState } from "react";
+import { useAppSelector } from "app/hooks";
+import {
+  selectConnectionTypeState,
+  setConnection,
+  setStep,
+} from "connection-type/connection-type.slice";
+import React, { useEffect, useRef, useState } from "react";
+import { useDispatch } from "react-redux";
 
+import Breadcrumb from "./Breadcrumb";
 import ConfigurationSettingsNav from "./ConfigurationSettingsNav";
 import { ConnectorParameters } from "./ConnectorParameters";
-import { CONNECTOR_PARAMETERS_OPTIONS } from "./constants";
-import DatasetConfiguration from "./sass/DatasetConfiguration";
-import { AddConnectionStep } from "./types";
+import { CONNECTOR_PARAMETERS_OPTIONS, STEPS } from "./constants";
+import DatasetConfiguration from "./DatasetConfiguration";
+import DSRCustomization from "./manual/DSRCustomization";
 
-type ConfigureConnectorProps = {
-  currentStep: AddConnectionStep;
-  connectionOption: ConnectionOption;
-};
-
-const ConfigureConnector: React.FC<ConfigureConnectorProps> = ({
-  currentStep,
-  connectionOption,
-}) => {
-  const [selectedItem, setSelectedItem] = useState(
-    CONNECTOR_PARAMETERS_OPTIONS[0]
+const ConfigureConnector: React.FC = () => {
+  const dispatch = useDispatch();
+  const mounted = useRef(false);
+  const { connectionOption } = useAppSelector(selectConnectionTypeState);
+  const [steps, setSteps] = useState([STEPS[0], STEPS[1], STEPS[2]]);
```

Comment:
This seems a little error prone if a new step is added since this is used many places and index based but I do like how you've been able to make these steps very dynamic here.